### PR TITLE
Chore/workflow corrections

### PR DESCRIPTION
**Pull request by ribasushi**
A few things got missed, I think this should be all.

Most notably - verify_final failed as-written, but not before doing the download :(

### CODE REVIEW

File: b288702/keyring.asc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Created with running:
 
-    $ curl --silent https://raw.githubusercontent.com/filecoin-project/phase2-attestations/9014d0d57a03f9eb68297d9ee68994fb238b26e7/b288702/README.md|grep '^\['|cut -d ' ' -f 2|xargs -n 1 sh -c 'echo $0 && curl --silent --location $0|gpg --no-default-keyring --keyring ./keyring.gpg --import'
+    $ curl --silent https://raw.githubusercontent.com/filecoin-project/phase2-attestations/e9ec1d11ffe0a98b95199833b3ebd537e589bcbe/b288702/README.md|grep '^\['|cut -d ' ' -f 2|xargs -n 1 sh -c 'echo $0 && curl --silent --location $0|gpg --no-default-keyring --keyring ./keyring.gpg --import'
```
Comment (vmx) [inline b288702/keyring.asc:3]: Please don't change this, as it isn't true. It should mention the commit it was actually created with. Also the checksum won't match the one uploaded to S3 then.

**Review by vmx — CHANGES_REQUESTED** [review]

File: b288702/keyring.asc (hunk 1 above)
Comment (ribasushi) [reply b288702/keyring.asc:3]: Good point, this is one thing I did not verify when I was checking the GPG workflow. Will force-push without this change.

File: b288702/README.md [hunk 2]
```diff
@@ -32,19 +32,19 @@ You will check out the commit corresponding to the [filecoin-proofs-v5.1.1](http
...
 2. Download checksums file and verification script:
 
 ```bash
-$ curl -O https://raw.githubusercontent.com/filecoin-project/phase2-attestations/e486b55/b288702/verify_all.sh \
--O https://raw.githubusercontent.com/filecoin-project/phase2-attestations/e486b55/b288702/download_initial_generation_prereqs.sh \
--O https://raw.githubusercontent.com/filecoin-project/phase2-attestations/e486b55/b288702/generate_initial.sh \
--O https://raw.githubusercontent.com/filecoin-project/phase2-attestations/e486b55/b288702/download_prereqs_contrib.sh \
--O https://raw.githubusercontent.com/filecoin-project/phase2-attestations/e486b55/b288702/verify_contrib.sh \
--O https://raw.githubusercontent.com/filecoin-project/phase2-attestations/e486b55/b288702/verify_final.sh \
+$ curl -O https://raw.githubusercontent.com/filecoin-project/phase2-attestations/e9ec1d11ffe0/b288702/verify_all.sh \
```
[the remark this answers is not in the document]
Comment (ribasushi) [reply b288702/README.md:42]: Right... there seems to be no agreement on whether `git rev-parse --short` is reasonable in this context. It was a remnant from a previous set of diffs, I should have kept that discussion on slack ( ref: https://protocollabs.slack.com/archives/CRS9PSD53/p1597939811012500?thread_ts=1597939567.012000&cid=CRS9PSD53 )

I will force-push without this entire commit.

File: b288702/keyring.asc (hunk 1 above)
Comment (ribasushi) [reply b288702/keyring.asc]: @vmx removed https://github.com/ribasushi/phase2-attestations/commit/d832cd5809941054ec entirely, will redo the doc portion when writing the final prose for it once the process is final.

**Review by vmx — APPROVED** [review]

File: b288702/verify_all.sh [hunk 3]
```diff
@@ -67,6 +67,7 @@ for i in $(seq 1 $n); do
     fi
 done
 
+$dir/download_prereqs_final.sh $proof $sector_size
```
Comment (ribasushi) [inline b288702/verify_all.sh:70]: I don't believe you need that - since everything was generate earlier... right?

**Review by ribasushi — COMMENTED** [review]
I can't self-approve but looks good. Left one nit

**Pull request merged**